go/client/cmd_chat_send.go

```diff
@@ -112,6 +129,7 @@ func (c *CmdChatSend) ParseArgv(ctx *cli.Context) (err error) {
 	c.clearHeadline = ctx.Bool("clear-headline")
 	c.hasTTY = isatty.IsTerminal(os.Stdin.Fd())
 	c.nonBlock = ctx.Bool("nonblock")
+	c.ephemeralLifetime = ctx.Duration("exploding-lifetime")
```

Comment:
Kind of a bummer that the parser for this type cuts off at hours and doesn't support `7d` for "seven days". @malgorithms do you think it's worth us cutting a ticket for later to extend it?